### PR TITLE
feat(spruce-skill): Add example of requesting that Core show a toast

### PR DESCRIPTION
**Pull request by gcpantazis**
See https://github.com/sprucelabsai/ai-spruce-web/pull/144

core-toasts
toasts-from-skill

### CODE REVIEW

File: packages/react-heartwood-components/src/components/Toast/Toast.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ const ToastHeader = (props: HeaderProps) => {
 }
 
 export type Props = {
+	/** Unique ID for the toast */
+	id: string,
```
Comment (justinhillsjohnson) [inline packages/react-heartwood-components/src/components/Toast/Toast.js]: @gcpantazis Can this also be a number?

Comment (gcpantazis) [reply packages/react-heartwood-components/src/components/Toast/Toast.js]: 🔥thanks man, fixed

File: packages/spruce-skill/interface/pages/skill-views/example.js [hunk 2]
```diff
@@ -52,6 +52,31 @@ class TestSkillView extends React.Component<Props> {
 										}
 									})
 								}}
+							/>{' '}
```
Comment (CarolynSM) [inline packages/spruce-skill/interface/pages/skill-views/example.js]: what's this space here for? I think you'll want to wrap these buttons in a `<ButtonGroup>` to space the buttons appropriately

Comment (gcpantazis) [reply packages/spruce-skill/interface/pages/skill-views/example.js]: Sure, done

**Pull request merged**